pkg/app/piped/platformprovider/lambda/client.go

```diff
@@ -126,6 +126,9 @@ func (c *client) CreateFunction(ctx context.Context, fm FunctionManifest) error
 		}
 		input.Architectures = architectures
 	}
+	if fm.Spec.EphemeralStorage != nil && fm.Spec.EphemeralStorage.Size != 0 {
```

Comment:
```suggestion
	if fm.Spec.EphemeralStorage != nil {
```
This condition should be updated because the value `0` will be yielded on the validation step.